src/parcsr_ls/par_lr_interp.c

```diff
@@ -591,9 +576,7 @@ hypre_BoomerAMGBuildStdInterp(hypre_ParCSRMatrix  *A,
             {
                indx = ihat[i1];
                if (indx > -1)
-               {
                   ahat[indx] += A_diag_data[jj];
-               }
```

Comment:
Why are all of the open/close squirly braces being removed?  Didn't you just go through a big effort to put them in?